source/tutorials/write-update-documents.txt

```diff
@@ -0,0 +1,291 @@
+  information about using query filters, see :ref:`Filters <java-rs-update-filters>`.
+
+- An update document, which specifies the update operator (the kind of update to
+  perform) and the fields and values that should change. For more information
+  about update documents, see :ref:`Update Operators <java-rs-update-operators>`.
+
+.. _java-rs-update-filters:
+
+Filters
+~~~~~~~
+
+Each update method requires a **query filter**, which specifies the
+search criteria that determine which documents to select for updates. To facilitate the creation of filter objects, the
+driver provides the ``Filters``class that provides filter condition helper
+methods. 
+
+To view a list of ``Filters`` helpers, see the `Filters
+API documentation <{+api+}/mongodb-driver-core/com/mongodb/client/model/Filters.html>`__.
+For more information about query filters, see the 
+:manual:`Query Filter Documents section </core/document/#query-filter-documents>` in
+the MongoDB Server manual. 
+
+.. _java-rs-update-operators:
+
+Update Operators
+~~~~~~~~~~~~~~~~
+
+To change a field in a document, MongoDB provides update operators.
+To specify the modification to perform using the update operators,
+create an update document. To learn more about update operators, see
+:manual:`Update Operators </reference/operator/update/>` in the Server manual.
+
+To facilitate the creation of update documents, the driver
+provides the ``Updates`` helper class.
+
+.. important::
+
+   The ``id`` field is immutable, so you cannot change the value of the
+   ``id`` field in a document.
+
+Update One Document
+-------------------
+
+To update a single document from a MongoDB collection, call the ``updateOne()``
+method and pass in your query filter as the first argument. Then, pass in your
+update operators as the second argument to the ``updateOne()`` method. After you call the
+``updateOne()`` method, chain the ``subscribe()`` method and pass in a subscriber.
+
+The following example uses ``Updates.set()`` to update a document in the
+``restaurants`` collection with a ``name`` value of ``Bagels N Buns`` to have
+the name ``2 Bagels 2 Buns``.
+
+.. code-block:: java
+   :copyable: true
+
+    ObservableSubscriber<UpdateResult> updateOneSubscriber = new SubscriberHelpers.OperationSubscriber<>();
+    restaurants.updateOne(eq("name", "Bagels N Buns"), set("name", "2
+    Bagels 2 Buns")).subscribe(updateOneSubscriber);
+    updateOneSubscriber.await();
+
+Update Multiple Documents
+-------------------------
+
+To update multiple documents from a MongoDB collection, call the ``updateMany()``
+method and pass in your query filter. Then, pass in your
+update operators as the second argument to the ``updateMany()`` method. After you call the
+``updateMany()`` method, chain the ``subscribe()`` method and pass in a subscriber.
+
+The following example updates all documents in the ``restaurants`` collection
+with a ``borough`` value of ``"Brooklyn"`` to have a star value of 4:
```

Comment:
I: it seems like this code example updates the `cuisine` values rather than the `borough` values. You could reword to something like "The following example updates all documents in which the cuisine value is "Pizza" to store a cuisine value of "Pasta""